### DISCUSSION TITLE
I want to make an exception when `strictToken` is specified.

### DISCUSSION THREAD

**Discussion by k35o** [chakra-ui/panda#2598] [Q&A] [answered]
Currently, when `strictToken` is set, all config tokens only allow token values and prevent custom or raw CSS values. 

However, I would like to configure it so that only the sizes token allows custom or raw CSS values. Is this possible?

I know we can use `[]` to handle any string, but I don't want to use it.

**Comment by astahmer** [comment, marked as answer]
it can be done with a Panda plugin like the `strict-tokens-scope` [here](https://github.com/astahmer/pandabox?tab=readme-ov-file#pandaboxpanda-plugins)

**Reply by k35o** [reply]
@astahmer 
Oh, that's awesome. Thank you!
I'll give it a try.

**Reply by k35o** [reply]
It worked almost as I expected. Thank you.
